src/index.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('node:fs');
+const path = require('node:path');
+const { Client, Collection, GatewayIntentBits } = require('discord.js');
+const { token } = require('./config.json');
```

Comment:
I think is preferable to use a .env file instead that the config.json, here is a reference:

- https://discordjs.guide/creating-your-bot/#using-environment-variables